falcon/request.py

```diff
@@ -296,6 +296,9 @@ class Request(object):
             A dict of name/value cookie pairs.
             See also: :ref:`Getting Cookies <getting-cookies>`
 
+        referer (str): Value of Referer header, or ``None`` if
```

Comment:
Would you mind moving this up to be next to the other header property descriptions?